packages/backend/server/src/schema.gql

```diff
@@ -11,11 +11,6 @@ type BlobNotFoundDataType {
   spaceId: String!
 }
 
-enum ChatHistoryOrder {
```

Comment:
should not commit this change

Reply:
Why this file always appear! How to avoid this?

Reply:
backend module will enable based on env or config, in this case, define `COPILOT_OPENAI_API_KEY=1` and `COPILOT_FAL_API_KEY=1` to enable copilot module